Content.Server/_Starlight/Vampire/VampireSystem.Objectives.cs

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2024 Rinary <72972221+Rinary1@users.noreply.github.com>
+// SPDX-FileCopyrightText: 2025 Dreykor <160512778+Dreykor@users.noreply.github.com>
+// SPDX-FileCopyrightText: 2025 Dreykor <Dreykor12@gmail.com>
+// SPDX-FileCopyrightText: 2025 Dreykor <arguemeu@gmail.com>
+// SPDX-FileCopyrightText: 2025 GabyChangelog <agentepanela2@gmail.com>
+// SPDX-FileCopyrightText: 2025 coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Server.Objectives.Components;
+using Content.Server.Objectives.Systems;
+using Content.Shared.Objectives.Components;
+using Content.Shared.Vampire.Components;
+using Content.Shared.Vampire;
+
+namespace Content.Server.Vampire;
```

Comment:
```suggestion
namespace Content.Server._Starlight.Vampire;
```
faz isso pra todos os que tao dentro de _Starlight, vai precisar atualizar os using